addons/account/wizard/setup_wizards.py

```diff
@@ -79,7 +79,7 @@ class SetupBarBankConfigWizard(models.TransientModel):
         comodel_name='account.journal', inverse='set_linked_journal_id',
         compute="_compute_linked_journal_id",
         check_company=True,
-        domain=[('type', '=', 'bank'), ('bank_account_id', '=', False)])
+        domain=[('type', 'in', ('bank', 'credit')), ('bank_account_id', '=', False)])
```

Comment:
not sure it's needed :thinking:

Reply:
I think we should have to move the domain in the bank wizard view.
I mean, we don't use the field in the credit card view (why?) so we don't need domain but if you want to avoid someone using a credit card journal for the bank view, you should move the domain there.